src/test/java/racingcar/RacingGameTest.java

```diff
@@ -0,0 +1,43 @@
+package racingcar;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class RacingGameTest {
+
+    private RacingGame racingGame;
+
+    @BeforeEach
+    public void beforeEach() {
+        racingGame = new RacingGame();
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = {"a,b,c|3", "a|1", "a,b,c,d,e,f|6"}, delimiter = '|')
+    @DisplayName("자동차 이름 추가에 따른 자동차 생성 테스트")
+    public void initCarsWithNameTest(String input, int result) {
```

Comment:
```suggestion
    @ParameterizedTest(name = "자동차 이름 추가에 따른 자동차 생성 테스트")
    @CsvSource(value = {"a,b,c|3", "a|1", "a,b,c,d,e,f|6"}, delimiter = '|')
    public void initCarsWithNameTest(String input, int result) {
```

다른 방법으로는 이렇게도 테스트의 이름을 나타낼 수 있습니다!
참고사항으로 알려드립니다~